README.md

```diff
@@ -202,6 +202,19 @@ AuthCallback authCallback = new AuthCallback() {
 
 If you've followed the configuration steps, the authentication result will be redirected from the browser to your application and you'll receive it in the Callback.
 
+#### Token Validation
+The ID token received as part of this authentication flow is automatically verified following the [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html).
+
+If your Auth0 account is set up to use Custom Domain but has not yet migrated from the [legacy behavior](https://auth0.com/docs/private-cloud/private-cloud-migrations/migrate-private-cloud-custom-domains#background), you need to override the expected issuer to match your Auth0 domain before starting the authentication.
```

Comment:
> If your Auth0 account is set up to use Custom Domain

Feels like this should be "Custom Domains" as that's what the high-level product feature is. Also, can we link to https://auth0.com/docs/custom-domains?

If so, there are a few instances in the diff below that should change to match (they don't all need the link though).

Reply:
Also, does this only affect PSaaS? If so, we should clarify that here too.

Reply:
I pushed some changes. LMK